README.md

```diff
@@ -121,39 +121,25 @@ The following tables lists the configurable parameters of the Orchestrate chart
-| `keyManager.nodeSelector`                | Node labels for pod assignment                                                                                         | `{}`        |
-| `keyManager.tolerations`                 | Tolerations for pod assignment                                                                                         | `[]`        |
-| `keyManager.affinity`                    | Affinity for pod assignment                                                                                            | `{}`        |
-| `keyManager.environment`                 | Environment variables passed to Orchestrate keyManager containers                                                      | `{}`        |
-| `keyManager.environmentSecrets`          | Environment variables (as Kubernetes secrets) passed to Orchestrate Key Manager containers                             | `{}`        |
-| `keyManager.existingSecret`              | If specified, extra environment variables will be added from externally created secret                                 | `nil`       |
-| `keyManager.migrate`                     | Run import keys from Vault formated with Orchestrate v2.5.X to Orchestrate Vault plugin (used in v21.1.X)              | `false`     |
-| `keyManager.vaultAgent.enabled`          | Run Vault Agent to retrieve the Key Manager client token from Hashicorp Vault                                          | `false`     |
-| `keyManager.vaultAgent.role`             | Role eligible to retrieve a token                                                                                      | `client`    |
-| `keyManager.vaultAgent.image.repository` | Hashicorp Vault image                                                                                                  | `vault`     |
-| `keyManager.vaultAgent.image.tag`        | Hashicorp Vault tag                                                                                                    | `1.6.2`     |
-| `keyManager.vaultAgent.config.wrapTTL`   | Response-wrapped TTL, see https://www.vaultproject.io/docs/agent/autoauth                                              | ``          |
+| `qkm.enabled`                     | Deploy Quorum Key Manager                           | `true`      |
+| `qkm.url`                     | Quorum Key Manager url, if not provided will be `fullname`.`namespace`                         | ""      |
+| `qkm.fullname`                     | Name                      | `quorumkeymanager`      |
+| `qkm.orchestrate.storeName`                     | Store name of orchestrate keys in the key manager                        | "orchestrate-eth"      |
+| `qkm.orchestrate.apiKey`                     | Secret value (base64 encoded) of the apiKey to authenticate orchestrate with key manager Manager                                     | `YWRtaW4tdXNlcg==`      |
```

Comment:
You have to add TLS auth option to

Reply:
#KEY_MANAGER_CLIENT_TLS_KEY=/certificates/client.key
#KEY_MANAGER_CLIENT_TLS_CERT=/certificates/client.crt